modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/DatasetTagger.tsx

```diff
@@ -0,0 +1,1402 @@
+            if (!activeHEDSchemas[hedTag.schemaName.toUpperCase()])
+              return null;
+
+            return (
+              <option
+                key={`${hedTag.name}-${onlyParentNodes ? 0 : 1}`}
+                id={hedTag.name}
+                value={hedTag.name}
+                label={
+                  onlyParentNodes
+                    ? `Schema: ${hedTag.schemaName.toUpperCase()}`
+                    : `[${hedTag.schemaName.toUpperCase()}] ${hedTag.longName}`
+                }
+                title={hedTag.longName}
+              />
+            )
+          })
+        }
+      </datalist>
+    );
+  }
+
+  const buildColumnNames = () => {
+    return columnOptions.filter((column) => {
+      return column.value !== 'channels'; // Removed until recognized
+    }).map((column) => {
+      const columnName = column.value;
+      const columnIsDirty = isDirty(columnName, '');
+
+      return (
+        <option
+          key={`column-name-${columnName}`}
+          value={columnName}
+        >
+          {columnName + (columnIsDirty ? ' (edited)' : '')}
+        </option>
+      )
+    });
+  }
+
+  const buildFieldValues = (columnName: string) => {
+    const column = columnOptions.find((opt) => {
+      return opt.value === columnName
+    });
+
+    if (!column)
+      return null;
+
+    return column.field_values.map((fieldValue) => {
+      const valueIsDirty = isDirty(columnName, fieldValue);
+      return (
+        <option
+          key={`field-value-${fieldValue}`}
+          value={fieldValue}
+          style={{
+            color: valueIsDirty ? '#F0AD4E' : 'black',
+          }} // Color instead of background due to unexpected behaviour
+        >
+          {fieldValue + (valueIsDirty ? ' (edited)' : '')}
+        </option>
+      )
+    })
+  }
+
+  const handleHEDMouseEnter = (hedSchemaElement: HEDSchemaElement) => {
+    if (hedSchemaElement) {
+      setDatasetTooltip({
+        title: hedSchemaElement.longName,
+        description: hedSchemaElement.description,
+     });
+    }
+  }
+
+  const handleHEDMouseLeave = () => {
+    clearDatasetTooltip();
+  }
+
+  const clearDatasetTooltip = () => {
+    setDatasetTooltip({
+      title: '',
+      description: '',
+    });
+  }
+
+  const getGroupedTagPairings = (hedTags: HEDTag[]) => {
+    const tagPairings = [];
+    if (activeFieldValue) { // null between transitions
+      hedTags.forEach((hedTag) => {
+        let pairRelID = hedTag.PairRelID;
+        while (pairRelID !== null) {
+          const pairRelTag = applyOverrides([...addedTags, ...datasetTags[activeColumnName][activeFieldValue]])
+            .find((t) => {
+              return t.ID === pairRelID;
+            });
+          if (pairRelTag) {
+            tagPairings.push(pairRelTag);
+            pairRelID = pairRelTag.PairRelID;
+          } else {
+            console.log(`Something went wrong. Tag with ID ${pairRelID} not found. Should not proceed.`);
```

Comment:
There should no be any console.logs. They are used for debugging and flagged by `make checkstatic`.

This looks like it's an error not a log.